src/databricks/labs/remorph/resources/queries/reconcile/installation/main.sql

```diff
@@ -3,10 +3,10 @@ CREATE TABLE IF NOT EXISTS main (
     recon_id STRING NOT NULL,
     source_type STRING NOT NULL,
     source_table STRUCT<
-                         catalog: STRING NOT NULL,
+                         catalog: STRING,
                          schema: STRING NOT NULL,
                          table_name: STRING NOT NULL
-                        > NOT NULL,
```

Comment:
Why `NOT NULL` constraint is removed for `source_table`?
For `reconciliation` we expect both `source_table` and `target_table` details, isn't it?